Sources/SwiftSyntax/RawSyntax.swift

```diff
@@ -736,9 +751,20 @@ extension RawSyntax {
     arena: SyntaxArena
   ) -> RawSyntax {
     validateLayout(layout: layout, as: kind)
+    var recursiveFlags = RecursiveRawSyntaxFlags()
```

Comment:
I consider this is a memberwise initializer.
Instead of calculating this in this function, please add `recursiveFlags: RecursiveRawSyntaxFlags` parameter to this, and edit  `makeLayout(arena:kind:uninitializedCount:initializingWith:)` and  `makeEmptyLayout(arena:kind:)`

Reply:
👍 Done